segmentation_processor_node:
  ros__parameters:
    input_segment_topic: "/ultralytics/segmentation/objects_segment"
    output_bbx3d_topic: "/darknet_ros_3d/bounding_boxes"
    point_cloud_topic: "/camera/points"
    debug_pointcloud_topic: "/segmentation_processor/debug_pointcloud"
    debug_markers_topic: "/segmentation_processor/debug_markers"
    # this software requires that the X-axes points directly to the scene.
    working_frame: "map"
    mininum_detection_threshold: 0.1
    minimum_probability: 0.35
    # interested_classes: ["orange","traffic light","chair","bottle","person", "italian_biscotti", "smoothie", "mango_juice", "crisps", "water", "sandwich", "toastie", "veggie_pot", "wrap", "espresso", "cappuccino", "americano","cup","table","bottle"]
    interested_classes: [ "truck", "tire"]
